OpenRA.Mods.Common/Widgets/Logic/Editor/MapToolsLogic.cs

```diff
@@ -42,10 +46,11 @@ public MapToolsLogic(Widget widget, World world, ModData modData, WorldRenderer
 
 			var markerToolPanel = widget.Get("MARKER_TOOL_PANEL");
 			toolPanels.Add(MapTool.MarkerTiles, markerToolPanel);
+			var mapGeneratorToolPanel = widget.Get<ScrollPanelWidget>("MAP_GENERATOR_TOOL_PANEL");
+			toolPanels.Add(MapTool.MapGenerator, mapGeneratorToolPanel);
 
 			toolsDropdown.OnMouseDown = _ => ShowToolsDropDown(toolsDropdown);
 			toolsDropdown.GetText = () => FluentProvider.GetMessage(toolNames[selectedTool]);
-			toolsDropdown.Disabled = true; // TODO: Enable if new tools are added
```

Comment:
it might be worth to disable the option if map generator was not defined in world

Reply:
Done, based on final tool count.